beginner_source/inference_on_aws_graviton_tutorial.rst

```diff
@@ -0,0 +1,343 @@
+                  Name    Self CPU %      Self CPU   CPU total %     CPU total   CPU time avg    # of Calls
+======================  =============  ============  ============  ============  ============  ============
+           aten::addmm        95.51%        5.821s        97.04%        5.914s      19.713ms           300
+       aten::clamp_min         2.33%     142.244ms         2.33%     142.244ms     711.220us           200
+           aten::copy_         1.51%      92.322ms         1.51%      92.322ms     307.740us           300
+     mymodel_inference         0.45%      27.713ms       100.00%        6.094s        6.094s             1
+          aten::linear         0.04%       2.495ms        97.16%        5.921s      19.736ms           300
+               aten::t         0.03%       2.131ms         0.07%       4.441ms      14.803us           300
+            aten::relu         0.03%       1.942ms         2.37%     144.186ms     720.930us           200
+======================  =============  ============  ============  ============  ============  ============
+
+``Self CPU time total: 6.094s``
+
+
+Following is the profiler output when run with the fast math mode enabled:
+
+``$ export DNNL_DEFAULT_FPMATH_MODE=BF16``
+
+.. table:: output with the fast math mode enabled
+
+======================  ============  ============  ============  ============  ============   =============
+                Name     Self CPU %      Self CPU    CPU total %   CPU total    CPU time avg    # of Calls
+======================  ============  ============  ============  ============  ============   =============
+           aten::addmm        93.31%        3.848s        95.66%        3.944s      13.148ms           300
+       aten::clamp_min         3.43%     141.309ms         3.43%     141.309ms     706.545us           200
+           aten::copy_         2.33%      95.916ms         2.33%      95.916ms     319.720us           300
+     mymodel_inference         0.67%      27.431ms       100.00%        4.123s        4.123s             1
+          aten::linear         0.06%       2.471ms        95.83%        3.951s      13.170ms           300
+               aten::t         0.05%       2.027ms         0.10%       4.243ms      14.143us           300
+            aten::relu         0.05%       1.928ms         3.47%     143.237ms     716.185us           200
+======================  ============  ============  ============  ============  ============   =============
+
+``Self CPU time total: 4.123s``
+
+
+This is around ``1.47x (4.123s vs 6.094s)`` improvement with fast math mode for the smaller batch dimensions. While the improvement is good, the performance is still not great. This is because of the runtime overhead (weights reorders and kernel launch time) from oneDNN and ACL backend outweighing the compute benefits from the ACL gemm kernels for the smaller batch compute.
```

Comment:
```suggestion
The fast math mode yields approximately a **1.47x  (4.123s vs 6.094s)**  for smaller batch dimensions. Although this improvement is noteworthy, the overall performance still leaves room for improvement. This is because of the runtime overhead (weights reorders and kernel launch time) from oneDNN and ACL backend outweighing the compute benefits from the ACL GEMM kernels for the smaller batch compute.
```